### DISCUSSION TITLE
Welcome to AndroidVeil Discussions!

### DISCUSSION THREAD

**Discussion by skydoves** [skydoves/AndroidVeil#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tom-dominic** [comment]
Hi @skydoves, Awesome work! Can we make below changes in the library?
- [ ] Can we change the root layout of `VeilRecyclerFrameView` from `RelativeLayout` to `ConstraintLayout `in order to get better performance?
- [ ] Can we provide a unique id to both `RecyclerView`(`userRecyclerView `& `veiledRecyclerView`) inside `VeilRecyclerFrameView` , so that we can unit test the UI by checking recycler view with the id

**Reply by elomonaco** [reply]
I second both these ideas.
